packages/cf.js/API_REFERENCE.md

```diff
@@ -0,0 +1,54 @@
+# API V0.0.2
+## `cf.js`
+
+- `Client`
+    - Properties
+        - `nodeProvider`
+    - Instance methods
+        - `getAppInfos(): AppInfo[]`
```

Comment:
I think the term `AppInfo` arose for some historic reason(s) that I'm not entirely sure about.

Any thoughts on renaming this to just `App`? Seems cleaner (i.e. `getApps()`)

Reply:
Argh but then the difference between App and AppInstance becomes confusing.

Reply:
Yeah, I remember there being some friction here. Moving on with `AppInfo`.

As for the terms, what's the rationale for an `AppInstance` not having it as a property, and for it needing to be fetched? IIUC an `AppInstance` is instantiated when the app actually gets installed, by which time the terms should be known and agreed to, and hence the state of the app instance should include this info. Is there some nuance I'm missing here?

Reply:
Earlier, when I said the only difference between an `AppInstance` and `AppInfo` is the terms, I was wrong. `AppInfo` also has the app's `name`, which is necessary for the playground to display in its 'active apps' list. The playground may want other manifest.json data too, such as an icon or description. If we make `AppInstances` and `AppInfos` the same thing, we'll probably want to include some sort of reference to the app's manifest in its properties.